edx-platform/pearson-vue-theme/lms/student-account/form_field.underscore

```diff
@@ -133,6 +133,6 @@
     <% } %>
 
     <% if( form === 'login' && name === 'password' ) { %>
-        <button type="button" class="forgot-password field-link"><%- gettext("Need help logging in?") %></button>
+        <button onclick="changeTitle()" type="button" class="forgot-password field-link"><%- gettext("Need help logging in?") %></button>
```

Comment:
Does this button has another event to show the forgot-password modal?. If yes, can we call `changeTitle()` function inside it?  so we don't need to add an extra onclick event in the button.  @luismorenolopera

Reply:
the event is called in another file, because js code does not work well in .underscore files @ivanvgh

Reply:
I understand, is good to know that, thanks.